operator/backrest/restore.go

```diff
@@ -323,7 +323,7 @@ func CreateRestoredDeployment(restclient *rest.RESTClient, cluster *crv1.Pgclust
 	cluster.Spec.UserLabels[config.LABEL_PG_CLUSTER] = cluster.Spec.ClusterName
 
 	// Set the Patroni scope to the name of the primary deployment.  Replicas will get scope using the
-	// 'current-primary' label on the pgcluster
+	// 'current-primary' annotation on the pgcluster
```

Comment:
? Is this comment accurate? This appears to be using the `UserLabels` for this operation, and this is part of the current HA implementation

Reply:
Chatted with Andrew about this one. After discussing, it seems the original comment likely refers to how the old HA solution (pre-4.2) needed to track the current primary. As of 4.2 it's now the scope each node needs to be aware of. 

The comment is now updated to reflect the current behavior, as follows:
```
// Set the Patroni scope to the name of the primary deployment.  Replicas will get scope using the
// 'crunchy-pgha-scope' label
```